packages/interaction-kit/src/definitions/application-commands.ts

```diff
@@ -3,172 +3,198 @@
-	id: Snowflake;
-	type: ApplicationCommandType;
-	name: string;
-	resolved?: ApplicationCommandInteractionDataResolved;
-	options?: ApplicationCommandInteractionDataOption[];
-	custom_id?: string;
-	component_type?: ComponentType;
-	target_id?: Snowflake;
-	values?: Array<SelectOption["value"]>;
-};
-
-/** @link https://discord.com/developers/docs/interactions/slash-commands#interaction-object-application-command-interaction-data-resolved-structure */
-export type ApplicationCommandInteractionDataResolved = {
-	users?: Record<Snowflake, User>;
-	members?: Record<Snowflake, Omit<GuildMember, "user" | "deaf" | "mute">>;
-	roles?: Record<Snowflake, Role>;
-	channels?: Record<
-		Snowflake,
-		Pick<Channel, "id" | "name" | "type" | "permissions">
-	>;
-	messages?: Record<Snowflake, Message>;
-};
-
-/** @link https://discord.com/developers/docs/interactions/slash-commands#interaction-object-application-command-interaction-data-option-structure */
-// TODO: Mutually exclusive, also what is option type?
-export type ApplicationCommandInteractionDataOption = {
-	name: string;
-	type: ApplicationCommandOptionType;
-	value?: OptionType; // FIXME: This needs to be set correctly
-	options?: ApplicationCommandInteractionDataOption[];
-};
-
-// HACK: This is to fix typechecking
-export type OptionType = unknown;
-
-/** @link https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-response-structure */
-// TODO: Check?
-export type InteractionResponse = {
-	type: InteractionCallbackType;
-	data?: InteractionApplicationCommandCallbackData;
-};
-
-/** @link https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-callback-type */
-export enum InteractionCallbackType {
-	PONG = 1,
-	CHANNEL_MESSAGE_WITH_SOURCE = 4,
-	DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE = 5,
-	DEFERRED_UPDATE_MESSAGE = 6,
-	UPDATE_MESSAGE = 7,
-}
-
-/** @link https://discord.com/developers/docs/interactions/slash-commands#interaction-response-object-interaction-application-command-callback-data-structure */
-export type InteractionApplicationCommandCallbackData = {
-	tts?: boolean;
-	content?: string;
-	embeds?: Embed[];
-	allowed_mentions?: AllowedMentions;
-	flags?: number;
-	components?: Component[];
-};
-
-/** @link https://discord.com/developers/docs/interactions/slash-commands#message-interaction-object-message-interaction-structure */
-export type MessageInteraction = {
-	id: Snowflake;
-	type: InteractionRequestType;
-	name: string;
-	user: User;
-};
+ import type { Snowflake } from "./snowflakes";
+ import type { GuildMember } from "./guild-members";
+ import { User } from "./users";
+ import { AllowedMentions, Channel } from "./channels";
+ import { Embed } from "./embeds";
+ import { Role } from "./roles";
+ import { Component, ComponentType, SelectOption } from "./components";
+ import { Message } from "./messages";
+ import { ChannelType } from "./channels"
```

Comment:
Duplicate import from `./channels`.
![image](https://user-images.githubusercontent.com/67111271/142777518-59670283-2209-4e6d-adfd-b67b9c8e325f.png)

```suggestion
 import { AllowedMentions, Channel, ChannelType } from "./channels";
 import { Embed } from "./embeds";
 import { Role } from "./roles";
 import { Component, ComponentType, SelectOption } from "./components";
 import { Message } from "./messages";
```

Reply:
already fixed